source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -339,6 +383,805 @@ TEST_F(NiRFmxWLANDriverApiTests, TXPFromExample_FetchData_DataLooksReasonable)
+  const auto dsss_mod_acc_fetch_evm_per_chip_mean_trace_response = client::dsss_mod_acc_fetch_evm_per_chip_mean_trace(stub(), session, "", 10.0);
+  const auto dsss_mod_acc_fetch_constellation_trace_response = client::dsss_mod_acc_fetch_constellation_trace(stub(), session, "", 10.0);
+
+  EXPECT_WARNING(dsss_mod_acc_fetch_evm_response, EVM_CHIPS_MUST_BE_300_WARNING);
+  EXPECT_EQ(0.0, dsss_mod_acc_fetch_evm_response.rms_evm_mean());
+  EXPECT_EQ(0.0, dsss_mod_acc_fetch_evm_response.peak_evm80211_2016_maximum());
+  EXPECT_EQ(0.0, dsss_mod_acc_fetch_evm_response.peak_evm80211_2007_maximum());
+  EXPECT_EQ(0.0, dsss_mod_acc_fetch_evm_response.peak_evm80211_1999_maximum());
+  EXPECT_EQ(0.0, dsss_mod_acc_fetch_evm_response.frequency_error_mean());
+  EXPECT_NE(0.0, dsss_mod_acc_fetch_evm_response.chip_clock_error_mean());
+  EXPECT_EQ(0, dsss_mod_acc_fetch_evm_response.number_of_chips_used());
+  EXPECT_WARNING(dsss_mod_acc_fetch_ppdu_information_response, EVM_CHIPS_MUST_BE_300_WARNING);
+  EXPECT_EQ(3, dsss_mod_acc_fetch_ppdu_information_response.data_modulation_format());
+  EXPECT_EQ(0, dsss_mod_acc_fetch_ppdu_information_response.payload_length());
+  EXPECT_EQ(1, dsss_mod_acc_fetch_ppdu_information_response.preamble_type());
+  EXPECT_EQ(0, dsss_mod_acc_fetch_ppdu_information_response.locked_clocks_bit());
+  EXPECT_EQ(0, dsss_mod_acc_fetch_ppdu_information_response.header_crc_status());
+  EXPECT_EQ(0, dsss_mod_acc_fetch_ppdu_information_response.psdu_crc_status());
+  EXPECT_WARNING(dsss_mod_acc_fetch_iq_impairments_response, EVM_CHIPS_MUST_BE_300_WARNING);
+  EXPECT_EQ(0.0, dsss_mod_acc_fetch_iq_impairments_response.iq_origin_offset_mean());
+  EXPECT_EQ(0.0, dsss_mod_acc_fetch_iq_impairments_response.iq_gain_imbalance_mean());
+  EXPECT_EQ(0.0, dsss_mod_acc_fetch_iq_impairments_response.iq_quadrature_error_mean());
+  EXPECT_WARNING(dsss_mod_acc_fetch_evm_per_chip_mean_trace_response, EVM_CHIPS_MUST_BE_300_WARNING);
+  EXPECT_EQ(0.0, dsss_mod_acc_fetch_evm_per_chip_mean_trace_response.x0());
+  EXPECT_EQ(1, dsss_mod_acc_fetch_evm_per_chip_mean_trace_response.dx());
+  EXPECT_EQ(0, dsss_mod_acc_fetch_evm_per_chip_mean_trace_response.evm_per_chip_mean_size());
+  EXPECT_EQ(0, dsss_mod_acc_fetch_evm_per_chip_mean_trace_response.evm_per_chip_mean().size());
+  EXPECT_WARNING(dsss_mod_acc_fetch_constellation_trace_response, EVM_CHIPS_MUST_BE_300_WARNING);
+  EXPECT_EQ(0, dsss_mod_acc_fetch_constellation_trace_response.actual_array_size());
+}
+
+// Disabled because it's getting the error during initiate of 
+// "-1074135024: \nIVI: (Hex 0xBFFA0010) Invalid value for parameter or property.\n\nProperty: IQ Rate\nRequested Value:  80.0e6\nValue Must Be Less Than or Equal to:  75.0e6\nDevice: Simulated_NI_PXIe_5601_Device_31\n\nStatus Code: -200077"
+TEST_F(NiRFmxWLANDriverApiTests, DISABLED_DSSSPowerRampFromExample_FetchData_DataLooksReasonable)
```

Comment:
I've added comments above all the currently Disabled tests. Each one fails with some error. I'm trying to figure out if it's something related to the simulated hardware setup and can be ignored or not.

I thought I'd put them up in case anyone had come across similar errors and see how we want to handle them.

Reply:
We should add a task or maybe better a bug on the feature to investigate these failures or delete them if we decide it isnt worth fixing.

Reply:
Added a Bug under the WLAN feature for the two Disabled tests: AB#1830352 to track fixing or removing the tests.